fusillade-api.yml

```diff
@@ -30,10 +33,40 @@ paths:
           description: OK
         default:
           description: Unexpected error
-  /v1/oauth/authorize:
+  /.well-known/openid-configuration:
+    get:
+      summary: This endpoint is part of OIDC
+      tags:
+        - oauth
+      operationId: fusillade.api.oauth.serve_openid_config
+      responses:
+        200:
+          description: Returns the supported openid endpoints
+  /.well-known/jwks.json:
+    get:
+      summary: This endpoint is part of OIDC
+      tags:
+        - oauth
+      operationId: fusillade.api.oauth.serve_jwks_json
+      responses:
+        200:
+          description: Returns the well know jwks
+  /cb:
```

Comment:
Internal routes should not be documented. You can use a Chalice route instead of a Connexion route to plumb them.

Reply:
will remove